### PR TITLE
Import cluster is failing when one or more storage nodes does not have at least one lvm

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
bugzilla: 1683603
tendrl-bug-id: Tendrl/gluster-integration#715

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/gluster_integration/sds_sync/brick_utilization.py [hunk 1]
```diff
@@ -59,18 +59,30 @@ def get_lvs():
             {"message": str(err)}
         )
         return None
-    out = out.split('\n')
```
Comment (GowthamShanmugam) [inline tendrl/gluster_integration/sds_sync/brick_utilization.py]: out is u'' when no lvm in storage node

**Comment by GowthamShanmugam** [comment]
@shtripat @nthomas-redhat @dahorak please review

File: tendrl/gluster_integration/sds_sync/brick_utilization.py (hunk 1 above)
Comment (shtripat) [reply tendrl/gluster_integration/sds_sync/brick_utilization.py]: So all we are doing here is checking non empty for `out` and adding a exception handling for parsing the `out` variable.
Looks good to me. Make sure this time we test all the possible scenarios.

Comment (GowthamShanmugam) [reply tendrl/gluster_integration/sds_sync/brick_utilization.py]: ok

**Comment by shtripat** [comment]
The pep8 issue is related to this PR or generic one?

**Comment by GowthamShanmugam** [comment]
@shtripat I verified this fix

**Comment by dahorak** [comment]
run build

**Pull request merged**